caravel/views.py

```diff
@@ -1219,14 +1256,12 @@ def favstar(self, class_name, obj_id, action):
     @has_access
     @expose("/slice/<slice_id>/")
     def slice(self, slice_id):
```

Comment:
I'm not sure of what the implications in terms of access rights/perms are here, I'd rather just keep `def explore`